### PR TITLE
Benchmarks - Support T4 and A10 in GEMM benchmark

### PR DESCRIPTION
**Pull request by abuccts**
Support T4 and A10 in GEMM benchmark.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def __init__(self, name, parameters=''):
                 'int4_tc': 'cutlass_tensorop_s4_i16864gemm_s4_256x128_128x3_*',
             }
         }
+        self.__kernel_map[7.5] = {k: self.__kernel_map[7.0][k] for k in self.__kernel_map[7.0] if 'fp64' not in k}
```
Comment (guoshzhao) [inline superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py:47]: Devices for inference don't  support fp64 kernels?

**Review by guoshzhao — APPROVED** [review]

File: superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py (hunk 1 above)
Comment (cp5555) [reply superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py:47]: Yes. @abuccts should write a comment for this.

Comment (abuccts) [reply superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py:47]: [Reference](https://images.nvidia.com/aem-dam/en-zz/Solutions/geforce/ampere/pdf/NVIDIA-ampere-GA102-GPU-Architecture-Whitepaper-V1.pdf) for A10,
> The GA102 GPU also features 168 FP64 units (two per SM), which are not depicted in this
diagram. The FP64 TFLOP rate is 1/64th the TFLOP rate of FP32 operations. The small number of FP64
hardware units are included to ensure any programs with FP64 code operate correctly

Comment (cp5555) [reply superbench/benchmarks/micro_benchmarks/cuda_gemm_flops_performance.py:48]: Great!

**Pull request merged**